atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -46,6 +46,25 @@ def __init__(self, tasker, workflow, hub, target=None, koji_ssl_certs_dir=None):
             }
         self.xmlrpc = create_koji_session(hub, koji_auth_info)
 
+    def get_patched_release(self, original_release, increment=False):
+        if original_release.count('.') > 1:
+            release, suffix, remainder = original_release.split('.', 2)
+        else:
+            remainder = ''
+            if original_release.count('.') == 1:
+                release, suffix = original_release.split('.', 1)
+            else:
+                release = original_release
+                suffix = ''
+
+        result = [str(int(release) + 1)] if increment else [release]
```

Comment:
Reworked code from https://github.com/projectatomic/atomic-reactor/pull/769#discussion_r132243638 a bit - renamed `increment` to `force_increment` and added the testcase